add cloud cli links (#4179)
fishtown-analytics/docs.getdbt.com: website/docs/docs/running-a-dbt-project/run-your-dbt-projects.md

title: "Run your dbt projects"
id: "run-your-dbt-projects"
---
You can run your dbt projects with [dbt Cloud](/docs/cloud/about-cloud/dbt-cloud-features) or [dbt Core](https://github.com/dbt-labs/dbt-core):

> - **dbt Cloud**: A hosted application where you can develop directly from a web browser using the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud). It also natively supports developing using a command line, [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). Among other features, dbt Cloud provides: 

Reviewer: ```suggestion
- **dbt Cloud**: A hosted application where you can develop directly from a web browser using the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud). It also natively supports developing using a command line interface, [dbt Cloud CLI](/docs/cloud/cloud-cli-installation). Among other features, dbt Cloud provides: 
```

  - Development environment to help you build, test, run, and [version control](/docs/collaborate/git-version-control) your project faster.
  - Share your [dbt project's documentation](/docs/collaborate/build-and-view-your-docs) with your team.
  - Integrates with the dbt Cloud IDE, allowing you to run development tasks and environment in the dbt Cloud UI for a seamless experience.

>   - Use the dbt Cloud CLI to develop and run dbt commands against your dbt Cloud development environment from your local command line.

Reviewer: Sugg finishing the "dbt Cloud provides..." and starting with a noun.
```suggestion
  - The dbt Cloud CLI to develop and run dbt commands against your dbt Cloud development environment from your local command line.
```

  - For more details, refer to [Develop in the Cloud](/docs/cloud/about-cloud-develop).

> - **dbt Core**: An open source project where you can develop from the command line.

Reviewer: We could also call out here that dbt Core is a "build your own" product.

> The dbt Cloud CLI and dbt Core are both command line tools that allow you to run your dbt projects and use the same dbt commands. The key difference is that the dbt Cloud CLI is designed to work specifically with dbt Cloud's infrastructure.

Author: The dbt Cloud CLI and dbt Core are both command line tools that enable you to run dbt commands. The key distinction is the dbt Cloud CLI is tailored for dbt Cloud's infrastructure and integrates with all its [features](/docs/cloud/about-cloud/dbt-cloud-features).

> The dbt Cloud CLI and dbt Core are both command line tools that allow you to run your dbt projects and use the same dbt commands. The key difference is that the dbt Cloud CLI is designed to work specifically with dbt Cloud's infrastructure.

Author: ```suggestion
The dbt Cloud CLI and dbt Core are both command line tools that enable you to run dbt commands. The key distinction is the dbt Cloud CLI is tailored for dbt Cloud's infrastructure and integrates with all its [features](/docs/cloud/about-cloud/dbt-cloud-features).
```
